app/src/main/java/org/simlar/service/SimlarServiceBroadcast.java

```diff
@@ -28,6 +28,8 @@
 
 public final class SimlarServiceBroadcast implements Serializable
 {
+	private static final long serialVersionUID = 1;
```

Comment:
We have the 'L' at two places in the code. However, there is another IntelliJ inspection helping to find all the other occurrences.

If we decide to go with 'L' everywhere, I would suggest to do it a pullrequest on its own which includes the new inspection.